src/main/java/org/pac4j/jax/rs/annotations/Pac4JSecurity.java

```diff
@@ -0,0 +1,42 @@
+package org.pac4j.jax.rs.annotations;
+
+import java.lang.annotation.Documented;
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+import org.pac4j.jax.rs.filter.SecurityFilter;
+
+/**
+ * 
+ * Identify the class or method as being filtered by {@link SecurityFilter}.
+ * 
+ * @author Victor Noel - Linagora
+ *
+ */
+@Target({ ElementType.METHOD })
+@Retention(RetentionPolicy.RUNTIME)
+@Documented
+public @interface Pac4JSecurity {
+
+    /**
+     * @return value for {@link SecurityFilter#setAuthorizers(String)}
+     */
+    String[] value() default {};
+
+    /**
+     * @return value for {@link SecurityFilter#setClients(String)}
+     */
+    String[] clients() default {};
+
+    /**
+     * @return value for {@link SecurityFilter#setMatchers(String)}
+     */
+    String[] matchers() default {};
+
+    /**
+     * @return value for {@link SecurityFilter#setMultiProfile(boolean)}
+     */
+    boolean multiProfile() default false;
```

Comment:
Boolean `null` means that I didn't configure anything. While `false` means it's `false` and we don't know if it's the default value and a defined value.

By using `null` and objects (and not primitives), the idea is to let the logics handle the default values like https://github.com/pac4j/pac4j/blob/master/pac4j-core/src/main/java/org/pac4j/core/engine/DefaultApplicationLogoutLogic.java#L40

As `null` values are not accepted, it looks like there is finally no debate.